grades/admin.py

```diff
@@ -0,0 +1,42 @@
+"""
+Admin for the grades app
+"""
+from django.contrib import admin
+
+from grades import models
+from micromasters.utils import get_field_names
+
+
+class FinalGradeAdmin(admin.ModelAdmin):
+    """Admin for FinalGradeA"""
+    model = models.FinalGrade
+    list_display = ('id', 'grade', 'user', 'course_run', )
+    ordering = ('course_run',)
+
+    def has_delete_permission(self, *args, **kwargs):  # pylint: disable=unused-argument
+        return False
+
+    def save_model(self, request, obj, form, change):
+        """
+        Saves object and logs change to object
+        """
+        obj.save_and_log(request.user)
+
+
+class FinalGradeAdminAuditAdmin(admin.ModelAdmin):
```

Comment:
Can we rename this to `FinalGradeAuditAdmin`, just to remove the redundant `Admin`?